elements/pf-dropdown/pf-dropdown.ts

```diff
@@ -0,0 +1,248 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { FloatingDOMController } from '@patternfly/pfe-core/controllers/floating-dom-controller.js';
+import styles from './pf-dropdown.css';
+import { bound } from '@patternfly/pfe-core/decorators/bound.js';
+import { ComposedEvent } from '@patternfly/pfe-core';
+import { query } from 'lit/decorators/query.js';
+import { queryAssignedElements } from 'lit/decorators/query-assigned-elements.js';
+
+interface MyCustomElement extends HTMLElement {
```

Comment:
let's see if we can do without this interface and these __divider and __disabled props